frontend/www/js/omegaup/components/arena/Course.vue

```diff
@@ -344,6 +336,26 @@ export default class ArenaCourse extends Vue {
     this.$emit('submit-run', { ...run, problem: this.activeProblem });
   }
 
+  onRunAdminDetails(guid: string): void {
+    this.$emit('show-run', {
+      request: { guid, isAdmin: this.isAdmin, hash: `#runs/show-run:${guid}/` },
```

Comment:
```suggestion
      request: { guid, isAdmin: this.isAdmin, hash: `#runs/show-run:${guid}` },
```

acá igual. los `/` al final sólo son para el [path](https://www.geeksforgeeks.org/components-of-a-url/), y esto es un fragmento.

Reply:
Listo.